crates/shared/src/price_estimation/competition.rs

```diff
@@ -66,11 +74,22 @@ impl PriceEstimating for CompetitionPriceEstimator {
 
 #[derive(Debug)]
 struct EstimateData<'a> {
+    kind: OrderKind,
     estimator_name: &'a str,
     estimate: Estimate,
     sell_over_buy: BigRational,
 }
 
+impl EstimateData<'_> {
+    /// Returns a label for the estimate's query kind.
+    pub fn kind_label(&self) -> &'static str {
+        match self.kind {
+            OrderKind::Buy => "buy",
+            OrderKind::Sell => "sell",
+        }
+    }
+}
+
```

Comment:
nit: Could make sense to implement something like `as_str` on `OrderKind` instead.

Reply:
I ended up adding a `OrderKind::label` method. I didn't go with `as_str` just because I wanted the method to reflect that it was meant to be used for metrics labels.